index.html

```diff
@@ -7,54 +7,64 @@
-                    <ul>
-                        <li>Bachelor's Degree in Social and Medical Psychology</br>
-                            2010-2016</br>
-                            Brest State Uneversity, Faculty of Social Pedagogy (Brest,Belarus).</li>
-                        <li>Master of Psychology Candidate Concetration: Social Psychology</br>
-                            Master's thesis: <em>Psychological preparedness of high-school students for volunteering activites.</em></br> 
-                            2017-2019</br>
-                            Brest State Uneversity, Faculty of Social Pedagogy (Brest,Belarus).</li>
-                    </ul>
-                <li>English level - <b>A2-Elementary</b>.</li>
-            </ol>        
-        </section>
+    <header class="header">
+        <div class="personal-information">
+            <h1>KUZMIN DZMITRY</h1>
+            <div class="objective">
+                <h3>Objective: Junior Front-End Developer</h3>
+            </div>
+            <div class="contact">
+                <img src="/images/icon-phone.svg" alt="phone">+372(33)323-17-35
+            </div>
+            <div class="contact">
+                 <a href="mailto:dimakuzmin9@gmail.com">
+                     <img src="/images/icon-email.svg" alt="email" >dimakuzmin9@gmail.com
+                </a>
+            </div>
+        </div>
+        <div class="avatar-container">
+                <img class="avatar" src="/images/photo-dima.jpg" alt="Kuzmin Dzmitry photo">
+        </div>
+    </header>
+    <div class="introduction">
+        <p>After working as a school psychologist, I decided that I didn't want to have anything with our political system.
+         And along with this, there was a need for self-determination. The desire to self-development and learn led me to the IT sphere.
+         I have no programming skills, but I have a great desire to become a competent specialist in this profession.
+        </p>
+    </div>
+    <div class="main-container">
+        <div class="skills">
+            <h3>Skills</h3>
+            <ul>
+                <li><span>easily teachable;</span></li>
+                <li><span>responsibility;</span></li>
+                <li><span>seIf-improvement;</span></li>
+            </ul>
+            <h3>Career</h3>
+            <p>Middle school No 9, Brest, Belarus.Position: Educational psychologist.</p>
+            <h5><u>Duties and responsibilites:</u></h5>
+            <ul>
+                <li><span>Psychological remidial activities with students;</span></li>
+                <li><span>Prevention and diagnostics of psyhological and mental health problems among students;</span></li>
+                <li><span>Documentation menedgment;</span></li>
+            </ul>
+        </div>
+        <div class="education">
+            <h3>Education</h3>
+            <ul>
+                <li><span>Bachelor's Degree in Social and Medical Psychology</br>
```

Comment:
The same about span and `<br>`. By the way, to escape from using `</br>` you can think of defining width for the list, so that the text is wrapped automatically.

Reply:
fixed